UI/index.html

```
@@ -3,7 +3,7 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Prepend Input Value</title>
+    <title>Prepend Input Value<title>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </title></title> ], start tag match failed [ <title> ] on line 6.

(tag-pair)